packages/attestation-service/src/sms/twilio.ts

```diff
@@ -76,15 +119,76 @@ export class TwilioSmsProvider extends SmsProvider {
     } catch (error) {
       throw new Error(`Twilio Messaging Service could not be fetched: ${error}`)
     }
+    if (this.verifyServiceSid) {
+      try {
+        await this.client.verify.services
+          .get(this.verifyServiceSid)
+          .fetch()
+          .then((service) => {
+            if (!service.customCodeEnabled) {
+              // Make sure that custom code is enabled
+              throw new Error(
+                'TWILIO_VERIFY_SERVICE_SID is specified, but customCode is not enabled. Please contact Twilio support to enable it.'
+              )
+            }
+          })
+      } catch (error) {
+        throw new Error(`Twilio Verify Service could not be fetched: ${error}`)
+      }
+    }
   }
 
   async sendSms(attestation: AttestationModel) {
-    const m = await this.client.messages.create({
-      body: attestation.message,
-      to: attestation.phoneNumber,
-      from: this.messagingServiceSid,
-      statusCallback: this.deliveryStatusURL,
-    })
-    return m.sid
+    // Prefer Verify API if Verify Service is present and
+    // user is not requesting a deeplink
+    if (this.verifyServiceSid && attestation.securityCode) {
```

Comment:
in what cases is the `attestation.securityCode` undefined? is there any way in the initialization step to initialize the Verify Service XOR the Messaging Service (instead of messaging + sometimes verify)?

Reply:
Good question. The security code may not be provided in the case that Valora requests signature (old method). I don't think there are any users on this version, but I didn't want to deprecate that feature in this PR.

Reply:
If we're certain that no one is using it, I can remove it in this PR. I just didn't have the time to investigate yet.

Reply:
Removed this logic after verifying with Valora team:
https://celo-org.slack.com/archives/CL7BVQPHB/p1626363263304300